spec/Section 6 -- Execution.md

```diff
@@ -103,8 +103,10 @@ Note: This algorithm is very similar to {CoerceArgumentValues()}.
 ## Executing Operations
 
 The type system, as described in the “Type System” section of the spec, must
-provide a query root object type. If mutations are supported, it must also
-provide a mutation root object type.
+provide a query root object type. If mutations/subscriptions are supported, it must also
+provide a mutation/subscription root object type, respectively.
```

Comment:
Probably should be plural:

> provide mutation/subscription root object types, respectively

However, I'd be in favor of splitting into two sentences, one about mutations and one about subscriptions.